### PR TITLE
Add release guideline

### PR DESCRIPTION
**Pull request by yunfengzhou-hub**
## What is the purpose of the change

This pull request adds the guideline to create a FeatHub release.

## Brief change log

- Adds the guideline to create a FeatHub release.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API: no

## Documentation

  - Does this pull request introduce a new feature? no
  - If yes, how is the feature documented? not applicable

### CODE REVIEW

**Comment by yunfengzhou-hub** [comment]
Hi @lindong28 could you please take a look at this PR?

File: docs/content/how-to/create-a-feathub-release.md [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+  Github Release.
+- The correctness of the source code and Python artifact have been verified and
+  the verification result is available to reviewers (e.g. as a Github Action
+  link).
+
+## Vote on the release candidate
+
+After the steps above, you should share the release candidiate with the Feathub
+repository admins to request their approval on this releast candidate. The
+sharing message should contain the following information:
```
Comment (lindong28) [inline docs/content/how-to/create-a-feathub-release.md:177]: Should we also provide reviewers the link to the release note?

File: docs/content/how-to/create-a-feathub-release.md [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+
+
+### Close milestone for current release
+
+The Github Milestone for the current release should be closed. No new issues
+should be linked to this milestone any longer.
+
+### Final checklist
+
+- Python artifacts released and indexed in the
```
Comment (lindong28) [inline docs/content/how-to/create-a-feathub-release.md:245]: Do we also need to check the result of "Git Release"?

**Review by lindong28 — COMMENTED** [review]
Thanks for the PR! Left two minor comments.

File: docs/content/how-to/create-a-feathub-release.md (hunk 1 above)
Comment (yunfengzhou-hub) [reply docs/content/how-to/create-a-feathub-release.md:177]: The release note should have been added to the Github Release created in steps above. I'll clarify this.

**Comment by yunfengzhou-hub** [comment]
Thanks for the comments @lindong28. I have updated the PR according to the comments.

**Comment by lindong28** [comment]
Thanks for the update. LGTM.

**Pull request merged**